pkg/cli/admin/upgrade/status/controlplane_test.go

```diff
@@ -230,15 +230,15 @@ func TestAssessControlPlaneStatus_Operators(t *testing.T) {
 					available(configv1.ConditionFalse).
 					progressing(configv1.ConditionTrue).operator,
 			},
-			expected: operators{Total: 2, Unavailable: 1},
+			expected: operators{Total: 2, Unavailable: 1, Upgrading: 1, Waiting: 1},
 		},
```

Comment:
Could we add a new test case to check the `Upgraded` field as well?

Reply:
Done